CONTRIBUTING

```diff
@@ -1,26 +1,27 @@
-HOW TO CONTRIBUTE PATCHES TO OpenSSL
+HOW TO CONTRIBUTE TO PATCHES OpenSSL
 ------------------------------------
 
 (Please visit https://www.openssl.org/community/getting-started.html for
 other ideas about how to contribute.)
 
-Development is coordinated on the openssl-dev mailing list (see the
+Development is coordinated on the openssl-project mailing list (see the
```

Comment:
No.  openssl-project is not intended to be a new openssl-dev.  We should say that we're coordinating development on github.